crypto/crypto.go

```diff
@@ -13,17 +13,29 @@ package crypto
 import (
 	"crypto/ed25519"
 	"crypto/rand"
+
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
 )
 
+// GenerateSecp256k1 generates a new secp256k1 private key.
+func GenerateSecp256k1() (*secp256k1.PrivateKey, error) {
```

Comment:
question: You only need this for testing right? not for any production code?

Reply:
Yes, I'm planning on adding a way to generate and store them in the keyring as well.